logic/atrium-logic-jvm/src/main/kotlin/ch/tutteli/atrium/logic/impl/DefaultPathAssertions.kt

```diff
@@ -147,6 +150,17 @@ class DefaultPathAssertions : PathAssertions {
             .build()
     }
 
+    private inline fun <T : Path> pathTypeAssertion(
```

Comment:
this is basically the same as fileTypeAssertion - can please you introduce a ioResultAssertion function which takes a `typeTest: (IOResult<BasicFileAttribute>) -> Boolean` can call it from file... and pathTypeAssertion